Libraries/Animated/src/nodes/AnimatedValue.js

```diff
@@ -175,6 +175,13 @@ class AnimatedValue extends AnimatedWithChildren {
   resetAnimation(callback?: ?(value: number) => void): void {
     this.stopAnimation(callback);
     this._value = this._startingValue;
+    // TODO(macOS ISS#2323203): A bug fix that hasn't been merged to upstream yet.
+    if (this.__isNative) {
+      NativeAnimatedAPI.setAnimatedNodeValue(
+        this.__getNativeTag(),
+        this._startingValue,
+      );
+    }
```

Comment:
nit: Didn't notice but I think we want to stop using ISS numbers and use GitHub issues instead to make em more public. Also there was a TODO [ and // ] END TODO corresponding marker. Sorry I missed that!